2021/25xxx/CVE-2021-25742.json

```
@@ -1,18 +1,111 @@
+                                        {
+                                            "version_affected": "<=",
+                                            "version_value": "1.0.0"
+                                        }, 
+                                        {
+                                            "version_affected": ">?",
+                                            "version_value": "1.0.0"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "Kubernetes"
+                }
+            ]
+        }
+    },
+    "credit": [
+        {
+            "lang": "eng",
+            "value": "Mitch Hulscher"
+        }
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A security issue was discovered in ingress-nginx where a user that can create or update ingress objects can use the custom snippets feature to obtain all secrets in the cluster."
             }
         ]
+    },
+    "generator": {
+        "engine": "Vulnogram 0.0.9"
+    },
+    "impact": {
+        "cvss": {
+            "attackComplexity": "LOW",
+            "attackVector": "NETWORK",
+            "availabilityImpact": "LOW",
+            "baseScore": 7.6,
+            "baseSeverity": "HIGH",
+            "confidentialityImpact": "HIGH",
+            "integrityImpact": "LOW",
+            "privilegesRequired": "LOW",
+            "scope": "UNCHANGED",
+            "userInteraction": "NONE",
+            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:L",
+            "version": "3.1"
+        }
+    },
+    "problemtype": {
+        "problemtype_data": [
+            {
+                "description": [
+                    {
+                        "lang": "eng",
+                        "value": "CWE-20: Improper Input Validation"
+                    }
+                ]
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "MLIST",
+                "url": "https://groups.google.com/g/kubernetes-security-announce/c/mT4JJxi9tQY"
+            },
+            {
+                "refsource": "CONFIRM",
+                "url": "https://github.com/kubernetes/ingress-nginx/issues/7837"
+            }
+        ]
+    },
+    "source": {
+        "defect": [
+            "https://github.com/kubernetes/ingress-nginx/issues/7837"
+        ],
+        "discovery": "EXTERNAL"
     }
```

Do you want to document the workaround (or solution?) here?
```suggestion
    },
    
    "work_around": [
        {
            "lang": "eng",
            "value": "This can be mitigated by disallowing snippet annotations on a supported version. Refer to https://github.com/kubernetes/ingress-nginx/issues/7837 for instructions."
        }
    ]
```